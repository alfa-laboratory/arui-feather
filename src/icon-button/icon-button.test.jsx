/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from 'react';
import { mount } from 'enzyme';

import IconButton from './icon-button';

describe('icon-button', () => {
    it('renders without problems', () => {
        let iconButton = mount(<IconButton />);
        expect(iconButton).toMatchSnapshot();
    });
});
