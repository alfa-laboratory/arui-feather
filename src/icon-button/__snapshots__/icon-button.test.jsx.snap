// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon-button renders without problems 1`] = `
<IconButton
  formNoValidate={false}
  size="m"
  tag="button"
  type="button"
>
  <button
    className="icon-button icon-button_size_m icon-button_theme_alfa-on-white"
    formNoValidate={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseOut={[Function]}
    onMouseUp={[Function]}
    role="button"
    type="button"
  />
</IconButton>
`;
